Kitodo/src/main/java/org/kitodo/dto/ClientDTO.java

```diff
@@ -38,6 +40,44 @@ public void setName(String name) {
         this.name = name;
     }
 
+    /**
+     * Gets users.
+     *
+     * @return The users.
+     */
+    public List<UserDTO> getUsers() {
+        return users;
+    }
+
+    /**
+     * Sets users.
+     *
+     * @param users
+     *            The users.
+     */
+    public void setUsers(List<UserDTO> users) {
+        this.users = users;
+    }
+
+    /**
+     * Gets usersSize.
+     *
+     * @return The usersSize.
+     */
+    public int getUsersSize() {
```

Comment:
Variable `usersSize` is defined as `Integer` but your method is returning primitive integer value. If value of `usersSize` is `NULL` then maybe a wrong value is returned. Maybe you should switch from `Integer` to `int` for the variable `usersSize` or do some other adjustment.